lib/wasmex.ex

```diff
@@ -92,6 +92,29 @@ defmodule Wasmex do
   - `:f32` a 32 bit float
   - `:f64` a 64 bit float
 
+  ### Linking
+
+  Wasm links may be given as an additional option.
+  Links is an array of module name-bytes pair.
+
+      iex> wasm = File.read!(TestHelper.wasm_link_test_file_path())
+      iex> linked_wasm = File.read!(TestHelper.wasm_test_file_path())
+      iex> links = [%{name: "utils", bytes: linked_wasm}]
+      iex> {:ok, pid} = Wasmex.start_link(%{bytes: wasm, links: links})
+      iex> Wasmex.call_function(pid, "sum_range", [1, 5])
+      {:ok, [15]}
```

Comment:
Nitpick alert - feel free to ignore or argue against my weak opinion here :)

naming is hard, I imaging someone reading the docs might have a hard time reading this when everything here is called a module or wasm bytes :D

maybe we can help a bit by using your utils example and name things as if it is actual code we would write outside of this example. Maybe name `linked_wasm` `utils_wasm`? (same below where `compiled_module` would be `utils_module`).